sources/packages/backend/libs/sims-db/src/entities/user.model.ts

```diff
@@ -7,8 +7,12 @@ import {
 } from "typeorm";
 import { ColumnNames, TableNames } from "../constant";
 import { BaseModel } from "./base.model";
-import { IdentityProviders } from "./identity-providers.type";
+import {
+  IdentityProviders,
```

Comment:
`IdentityProviders` should be removed and its occurrences replaced by `SpecificIdentityProviders`.

Reply:
Changed it as discussed.